pom.xml

```diff
@@ -114,6 +118,17 @@
 
 	<dependencies>
 
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-web</artifactId>
+			<exclusions>
+				<exclusion>
+					<groupId>org.apache.tomcat.embed</groupId>
+					<artifactId>tomcat-embed-core</artifactId>
+				</exclusion>
+			</exclusions>
```

Comment:
On later versions of javax.servlet-api the classesXssPreventionRequestWrapper.java, GzipBodyDecompressFilter.java, GzipResponseStream.java need some extra methods overridden.

Appropriate implementations will need to be added to allow Spring Boot's javax.servlet-api to be the default.

https://github.com/spring-projects/spring-boot/issues/8943#issue-222818336